src/PantheonShell.vala

```diff
@@ -243,7 +247,55 @@ namespace Gala {
             return;
         }
 
-        // TODO
+        Meta.Side anchor_side = TOP;
+        switch (anchor) {
```

Comment:
Not sure whether we can just cast this or whether mutters enum order is likely to change